fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -558,8 +558,8 @@ private Loan(final String accountNo, final Client client, final Group group, fin
         this.createStandingInstructionAtDisbursement = createStandingInstructionAtDisbursement;
 
         /*
-         * During loan origination stage and before loan is approved principal_amount, approved_principal and
-         * principal_amount_demanded will same amount and that amount is same as applicant loan demanded amount.
+         * During loan origination stage and before loan is approved principal_amount, approved_principal and principal_amount_demanded will same amount and that amount is same as applicant loan
```

Comment:
Please revert these unnecessary changes. I'm fine with javadoc changes if needed but these are polluting the PR.